oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/worker/MetricsPersistentWorker.java

```diff
@@ -395,6 +397,18 @@ private Metrics requireInitialization(Metrics metrics) {
         return null;
     }
 
+    @Override
+    public void onServerBooted(final BootingStatus bootingStatus) {
+        timeOfLatestStabilitySts = TimeBucket.getMinuteTimeBucket(
```

Comment:
Please make `timeOfLatestStabilitySts` `volatile`, now it's updated in multiple threads

Reply:
> Please make `timeOfLatestStabilitySts` `volatile`, now it's updated in multiple threads

Or change the type to `AtomicLong` (as well as the one in class `ClusterStatus`)